cypress/integration/pages/liveRadio/helper.js

```diff
@@ -1,5 +1,20 @@
+import envConfig from '../../../support/config/envs';
+
 // the externalId `bbc_oromo_radio` is overriden to `bbc_afaanoromoo` in production code
 const getMappedServiceId = externalId =>
   externalId === 'bbc_oromo_radio' ? 'bbc_afaanoromoo_radio' : externalId;
 
-export default getMappedServiceId;
+export const getEmbedUrl = (body, language) => {
+  const { id, externalId } = body.content.blocks[2];
+  const mediaId = getMappedServiceId(externalId);
```

Comment:
This is the service id (eg bbc_arabic_radio) rather than the media id

Media id is "liveradio" in this case, or the episode id for on demand radio

```suggestion
  const serviceId = getMappedServiceId(externalId);
```

Reply:
Actually "service id" isn't correct either, its just an erroneous naming convention we have.  What we refer to as "service id" in code is generally what product/editorial refers to as "master brand".

Now is probably not the time to fix this, but strings like "arabic" should be called services, strings like "bbc_arabic_radio" should be called brands, and "liveradio" should be the media id

I believe product call the ser